CMakeLists.txt

```diff
@@ -5,15 +5,24 @@ if (NOT ANDROID AND NOT (TARGET SDL2 OR TARGET SDL2-static))
 	find_package(SDL2 REQUIRED)
 endif()
 
-option(SUPPORT_JPG "Support loading JPEG images" ON)
-option(SUPPORT_PNG "Support loading PNG images" ON)
-option(SUPPORT_WEBP "Support loading WEBP images" OFF)
-option(BUILD_SHOWIMAGE "Build the showimage sample program" OFF)
+option(SDL_IMAGE_SUPPORT_JPG "Support loading JPEG images" ON)
+option(SDL_IMAGE_SUPPORT_PNG "Support loading PNG images" ON)
+option(SDL_IMAGE_SUPPORT_WEBP "Support loading WEBP images" OFF)
+option(SDL_IMAGE_BUILD_SHOWIMAGE "Build the showimage sample program" OFF)
+option(SDL_IMAGE_USE_WIC "Use WIC Backend on Windows" OFF)
 option(BUILD_SHARED_LIBS "Build the library as a shared library" ON)
 if (APPLE)
 	option(ENABLE_APPLE_IMAGEIO "Use native Mac OS X frameworks for loading images" ON)
 endif()
 
+if (NOT WIN32 AND SDL_IMAGE_USE_WIC)
+	message(FATAL_ERROR "WIC is only supported on Windows")
+endif()
+if (NOT SDL_IMAGE_SUPPORT_PNG OR NOT SDL_IMAGE_SUPPORT_JPG)
+	if (SDL_IMAGE_USE_WIC)
+		message(FATAL_ERROR "WIC requires PNG and JPG support.")
```

Comment:
This is wrong. If you really want, you can add to the description that this option provides builtin support for jpg, png and tiff.

Reply:
Oh, that's right. I forgot it. However, I think that the maintain for using WIC is used to reduce third-party dependency, in this library, we don't seem to need to add a new option for tiff format — 
it will not be compiled, when using WIC.